test/hotspot/jtreg/runtime/jni/getCreatedJavaVMs/exeGetCreatedJavaVMs.c

```diff
@@ -91,7 +91,11 @@ int main (int argc, char* argv[]) {
   pthread_t threads[NUM_THREADS];
   for (int i = 0; i < NUM_THREADS; i++ ) {
     printf("[*] Creating thread %d\n", i);
-    int status = pthread_create(&threads[i], NULL, thread_runner, (void *)(intptr_t)i);
+    pthread_attr_t attr;
+    pthread_attr_init(&attr);
+    size_t stack_size = 0x100000;
+    pthread_attr_setstacksize(&attr, stack_size);
+    int status = pthread_create(&threads[i], &attr, thread_runner, (void *)(intptr_t)i);
```

Comment:
In real world app, we should use `pthread_attr_destroy(&attr)` after we no longer need it, but in this test it's no big deal (hopefully anyone reading this code will not stop here and learn to leak that resource).

Reply:
Yes including the destroy would be good code hygiene.

Reply:
Thanks @gerard-ziemski and @dholmes-ora for the suggestion. I have included `pthread_attr_destroy(&attr)`